sql/core/src/main/scala/org/apache/spark/sql/catalyst/dynamicpruning/CleanupDynamicPruningFilters.scala

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.spark.sql.dynamicpruning
+package org.apache.spark.sql.catalyst.dynamicpruning
```

Comment:
let's move it to execution? it's also where AQE landed.

Reply:
Copy that.